spec/models/user_spec.rb

```diff
@@ -5,9 +5,54 @@
   include TestFactories
 
```

Comment:
I gave it a try but my specs don't pass after swapping create with
build/new. My guess is, there's probably something missing in my
declaration or it might not make sense to stub out these objects since
we're testing them but let's talk more about it tomorrow.

On Mon, Jul 6, 2015 at 7:41 PM Bobbilee <notifications@github.com> wrote:

> In spec/models/user_spec.rb
> <https://github.com/voltaaage/Bloccit/pull/25#discussion_r33998944>:
>
> > @@ -5,9 +5,54 @@
> >    include TestFactories
> >
>
> just meant overall in all ur specs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/voltaaage/Bloccit/pull/25/files#r33998944>.
>